mlflow/keras.py

```diff
@@ -73,16 +105,16 @@ def log_model(keras_model, artifact_path, **kwargs):
     >>>   mlflow.keras.log_model(keras_model, "models")
     """
     Model.log(artifact_path=artifact_path, flavor=mlflow.keras,
-              keras_model=keras_model, **kwargs)
+              keras_model=keras_model, conda_env=conda_env, **kwargs)
 
 
 def _load_model(model_file):
     import keras.models
     import h5py
     # NOTE: Keras 2.2.3 does not work with unicode paths in python2. Pass in h5py.File instead of
     # string to avoid issues.
-    model_file = h5py.File(os.path.abspath(model_file),)
-    return keras.models.load_model(model_file)
+    with h5py.File(os.path.abspath(model_file), "r") as model_file:
```

Comment:
Using a context manager here ensures that the h5py File is closed after the model is loaded. Previously, the file was not closed; this caused issues when attempting to serve them model using a multi-worker nginx configuration